langchain_community_placeholder/utilities/cassandra_database.py

```diff
@@ -109,34 +72,13 @@ def get_keyspace_schema_no_throw(self, keyspace: str) -> str:
             """Format the error message"""
             return f"Error: {e}"
 
-    def get_keyspace_schema(self, keyspace: str) -> str:
+    def get_keyspace_schema(self, keyspace: str) -> List[Table]:
         """Get the schema for the specified keyspace."""
+        if keyspace in self._keyspaces:
```

Comment:
I contemplated this a few times. Caching the keyspace and just not asking for it again. Getting the keyspace again is a pretty cheap operation if asked. IMO we should just always make the call and not introduce cache invalidation. WDYT?